docs/snippets/ngrok-push-secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: external-secrets.io/v1alpha1
+kind: PushSecret
+metadata:
+  name: ngrok-push-secret-example
+spec:
+  deletionPolicy: Delete
+  refreshInterval: 10m # Refresh interval for which push secret will reconcile
+  secretStoreRefs: # A list of secret stores to push secrets to
+    - name: ngrok # Must match SecretStore on the cluster
+      kind: SecretStore
+  selector:
+    secret:
+      name: SECRET_NAME # Source Kubernetes secret to be pushed
+  data:
+    - match:
+        secretKey: MY_K8S_SECRET_KEY # The key within the Kubernetes secret to push
+        remoteRef:
+          remoteKey: MY_NGROK_SECRET_NAME # The name of the secret in the ngrok vault
```

Comment:
Is this compatible with pushing the whole Secret? i.e. `secretKey: ""` ? If not, we should document this

Reply:
I could see it is supported from the client implementation. Can we add another `data` entry here commented out to server as example?